### PR TITLE
feat: add kitty items tutorial structure

### PR DESCRIPTION
**Pull request by alxflw**
Closes https://github.com/onflow/kitty-items/issues/220

## Description

This PR will add a new Kitty Items tutorial to the documentation.

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/content/kitty-items/run-testnet.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Run on testnet
+description: Run the project on the testnet
```
Comment (10thfloor) [inline docs/content/kitty-items/run-testnet.md:3]:
```suggestion
description: Run the project on testnet
```

File: docs/content/kitty-items/modify.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Modify
+description: Change up things
```
Comment (10thfloor) [inline docs/content/kitty-items/modify.md:3]:
```suggestion
description: Change things up
```

File: docs/content/kitty-items/get-started.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+> docker version
+
+[...]
+Server: Docker Engine - Community
+ Engine:
+  Version:          20.10.12
+[...]
+```
```
Comment (10thfloor) [inline docs/content/kitty-items/get-started.md:25]: This is going to go away soon. 
https://github.com/onflow/kitty-items/issues/214

File: docs/content/kitty-items/get-started.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+### NodeJS
+
+Ensure you have NodeJS [installed](https://nodejs.org/en/). Version **14+** is required.
+
+```
+> node -v
+
+v14.18.3
```
Comment (10thfloor) [inline docs/content/kitty-items/get-started.md:34]:
```suggestion
v14.x.x
```

File: docs/content/kitty-items/get-started.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+```sh
+git clone https://github.com/onflow/kitty-items.git
+```
+
+## Install dependencies
+
+- Run `npm install` in the root of the project.
+- Run `npx lerna exec npm install` to install project dependencies.
+
+**Congratulations, KittyItems is now all set up. Let's start the project locally next**
```
Comment (10thfloor) [inline docs/content/kitty-items/get-started.md:58]:
```suggestion
**Congratulations, KittyItems is set up. Next, let's start the project locally.**
```

File: docs/content/kitty-items/index.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+title: Kitty Items
+description: A CryptoKitties Sample App
+sidebar_title: Introduction
+---
+
+Kitty Items is a complete NFT marketplace modeled after CryptoKitties and built with [Cadence](/candece). Use it to learn how to develop on Flow and as a blueprint for your own marketplace.
```
Comment (10thfloor) [inline docs/content/kitty-items/index.md:7]:
```suggestion
Kitty Items is a complete NFT marketplace built with [Cadence](/cadence). Use it to learn how to develop on Flow and as a blueprint for your own marketplace.
```

File: docs/content/kitty-items/index.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+> [CryptoKitties](https://www.cryptokitties.co/) is Web3 game that put NFTs on the map. Now developers can make their own NFT marketplace influenced by CryptoKitties.
+
+Throughout this tutorial, Kitty Items will be used to help you understand the concepts behind the Flow blockchain and its programming language Cadence.
+
+After completing this tutorial, you will understand how to deploy smart contracts, mint NFTs, and integrate wallets with Flow’s Client Library.
```
Comment (10thfloor) [inline docs/content/kitty-items/index.md:13]:
```suggestion
After completing this tutorial, you will understand how to deploy smart contracts, mint NFTs, and integrate wallets with Flow’s client library (FCL).
```

File: docs/content/kitty-items/index.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+[![Kitty Items - Landing Page](landing-page.png)](http://kitty-items-flow-testnet-prod.herokuapp.com/)
+
+## Project overview
+
+The project consists of 3 key components:
+
+- Frontend: a static web application based on NextJS
+- Backend: an API based on NodeJS
+- Cadence: A set of contracts, scrips, and transactions
```
Comment (10thfloor) [inline docs/content/kitty-items/index.md:27]:
```suggestion
- Cadence: a set of contracts, scrips, and transactions
```

File: docs/content/kitty-items/index.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+Here is how it all comes together:
+
+![Project overview](kitty-items-diagram.png)
+
+### 1. Web App (Static website) | [kitty-items/web](https://github.com/onflow/kitty-items/tree/master/web)
+
+A true dapp, client-only web app. This is a complete web application built with React that demonstrates how to build a static website that can be deployed to an environment like IPFS and connects directly to the Flow blockchain using `@onflow/fcl`. No servers required. `@onflow/fcl` handles authentication and authorization of [Flow accounts](https://docs.onflow.org/concepts/accounts-and-keys/), [signing transactions](https://docs.onflow.org/concepts/transaction-signing/), and querying data using using Cadence scripts.
```
Comment (10thfloor) [inline docs/content/kitty-items/index.md:35]:
```suggestion
A true dapp, client-only web app. This is a complete web application built with NextJS and React that demonstrates how to build a static website that can be deployed to an environment like IPFS and connects directly to the Flow blockchain using `@onflow/fcl`. No servers required. `@onflow/fcl` handles authentication and authorization of [Flow accounts](https://docs.onflow.org/concepts/accounts-and-keys/), [signing transactions](https://docs.onflow.org/concepts/transaction-signing/), and querying data using using Cadence scripts.
```

File: docs/content/kitty-items/run-locally.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+title: Run locally
+description: Run the backend, frontend, and Flow emulator locally
+sidebar_title: Run locally
+---
+
+Continue reading the sections below for instructions on how to start the project for local development.
+
+## Local development
+
+1.  Run `npm run start:dev`
```
Comment (10thfloor) [inline docs/content/kitty-items/run-locally.md:11]:
```suggestion
1.  Run `npm run start:local`
```

**Comment by alxflw** [comment]
@10thfloor this should be ready for final review and merging

**Pull request merged**